app/mailboxes/inbox_mailbox.rb

```diff
@@ -50,7 +50,7 @@ def user_token
   def sending_user
     return @sending_user if @sending_user
 
-    if (user = User.where("mailing_list_mode > 0 AND mailing_list_token = ?", user_token).first) &&
+    if (user = User.where("mailing_list_token = ?", user_token).first) &&
```

Comment:
```suggestion
    if (user = User.find_by("mailing_list_token = ?", user_token)) &&
```

I'm a little surprised standardrb didn't make this change, I though it was one of its automatic refactors.

Reply:
```
$ standardrb app/mailboxes/inbox_mailbox.rb 
standard: Use Ruby Standard Style (https://github.com/standardrb/standard)
  app/mailboxes/inbox_mailbox.rb:53:21: Rails/WhereEquals: Use `where(mailing_list_token: user_token)` instead of manually constructing SQL.
standard: Run `standardrb --fix` to potentially fix one problem.
```

standardrb suggests it to be changed from manual string to a hash